esmvaltool/cmorizers/data/downloaders/datasets/gcp.py

```diff
@@ -0,0 +1,39 @@
+"""Script to download GCP from its webpage."""
+import logging
+import os
+
+from esmvaltool.cmorizers.data.downloaders.wget import WGetDownloader
+
+logger = logging.getLogger(__name__)
+
+
+# pylint: disable=invalid-name
+def download_dataset(config, dataset, _, __, overwrite):
+    """Download dataset.
+
+    Parameters
+    ----------
+    config : dict
+        ESMValTool's user configuration
+    dataset : str
+        Name of the dataset
+    start_date : datetime
+        Start of the interval to download
+    end_date : datetime
+        End of the interval to download
+    overwrite : bool
+        Overwrite already downloaded files
+    """
+    downloader = WGetDownloader(
+        config=config,
+        dataset=dataset,
+        overwrite=overwrite,
+    )
+    downloader.tier = 2
+    os.makedirs(downloader.local_folder, exist_ok=True)
+    downloader.download_file(
+        "https://data.icos-cp.eu/objects/OT_YY6iORypk2yAcwjMpUQpo",
+        wget_options=[
+            f"--output-document={downloader.local_folder}/"
+            "Global_Carbon_Budget_2018v1.0.xlsx"
+        ])
```

Comment:
While the path to file seems correct, the downloaded file is 4 times lighter than the original one (40 kb vs 161 kb) and not viewable (somehow corrupted). Could you please check?

Reply:
It is downloading a webpage with the license. I can't find a direct link to the file. 

This particular dataset is Creative Commons 4.0, so we can use our own link if we so choose

Reply:
Maybe @schlunma would know what to do about downloading this file for the GCP data?

Reply:
Removed the auto downloader. Maybe we can revisit it after this got merged